x-pack/dockerlogbeat/magefile.go

```diff
@@ -91,7 +100,7 @@ func Build() error {
 }
 
 // Clean removes working objects and containers
```

Comment:
nit: Please update the comment (`CleanDocker`)

Reply:
D'oh!